Makefile

```diff
@@ -29,76 +29,73 @@ jenkins: foc jenkins_build_dir
 # ================================================================
 # Genode toolchain. Only needs to be done once per target (x86/arm).
 toolchain:
-	wget https://nextcloud.os.in.tum.de/s/9idiw8BLbuwp35z/download -O toolchain
-	tar xfj toolchain -C .
+	wget -qO- https://nextcloud.os.in.tum.de/s/9idiw8BLbuwp35z/download | tar xj -C .
```

Comment:
Since Genode 16.08 we don't provide the libports ourselves anymore and the respective lines are commented out. We could add them again for 16.08 to improve download speed in our local network but otherwise we don't really gain anything from it. Especially since now the prepare_ports command can process multiple ports at once.

Reply:
Did a cleanup of the provision file and .gitignore @privatereese